src/components/flat-table/flat-table.pw.tsx

```diff
@@ -0,0 +1,3619 @@
+  });
+
+  test(`should render checkbox with ariaLabelledBy`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(<FlatTableColorRowSelectableComponent />);
+
+    await expect(
+      flatTableHeaderCells(page).nth(0).locator("input")
+    ).toHaveAttribute("aria-labelledby", CHARACTERS.STANDARD);
+  });
+
+  test(`should render with sticky header`, async ({ mount, page }) => {
+    await mount(
+      <div style={{ height: "150px" }}>
+        <FlatTableComponent hasStickyHead />
+      </div>
+    );
+
+    await expect(flatTable(page)).toHaveCount(1);
+    await expect(flatTable(page).locator("thead")).toHaveCSS(
+      "position",
+      "sticky"
+    );
+
+    for (let i = 0; i < 5; i++) {
+      if (i === 3 || i === 4) {
+        await expect(flatTableBodyRowByPosition(page, i)).not.toBeInViewport();
+      } else {
+        await expect(flatTableBodyRowByPosition(page, i)).toBeInViewport();
+      }
+    }
+
+    const wrapperElement = flatTableWrapper(page);
+    await wrapperElement.evaluate((wrapper) => wrapper.scrollBy(0, 150));
+
+    for (let i = 0; i < 5; i++) {
+      if (i === 0 || i === 1) {
+        await expect(flatTableBodyRowByPosition(page, i)).not.toBeInViewport();
+      } else {
+        await expect(flatTableBodyRowByPosition(page, i)).toBeInViewport();
+      }
+    }
+  });
+
+  test(`should render with sticky header and multiple rows`, async ({
+    mount,
+    page,
+  }) => {
+    await mount(
+      <div style={{ height: "150px" }}>
+        <FlatTableSpanComponent hasStickyHead />
+      </div>
+    );
+
+    await expect(
+      flatTableHeaderRowByPosition(page, 0).locator("th").first()
+    ).toHaveCSS("top", "0px");
+    await expect(
+      flatTableHeaderRowByPosition(page, 1).locator("th").first()
+    ).toHaveCSS("top", "40px");
+  });
+
+  test(`should render with sticky footer`, async ({ mount, page }) => {
+    await mount(<FlatTableFooterComponent hasStickyFooter />);
+
+    await expect(flatTablePager(page).locator("..")).toHaveCSS(
+      "position",
+      "sticky"
+    );
+
+    for (let i = 0; i < 5; i++) {
+      if (i === 4) {
+        await expect(flatTableBodyRowByPosition(page, i)).not.toBeInViewport();
+      } else {
+        await expect(flatTableBodyRowByPosition(page, i)).toBeInViewport();
+      }
+    }
+
+    const wrapperElement = flatTableWrapper(page);
+    await wrapperElement.evaluate((wrapper) => wrapper.scrollBy(1366, 150));
+    const pagerParent = flatTablePager(page).locator("..");
+    await pagerParent.scrollIntoViewIfNeeded();
+
+    await expect(flatTableHeaderCells(page).first()).not.toBeInViewport();
+    await expect(pagerParent).toBeInViewport();
+
+    for (let i = 0; i < 5; i++) {
+      if (i === 0) {
+        await expect(flatTableBodyRowByPosition(page, i)).not.toBeInViewport();
+      } else {
+        await expect(flatTableBodyRowByPosition(page, i)).toBeInViewport();
+      }
+    }
+  });
+
+  ([...colorThemes] as [
```

Comment:
```suggestion
colorThemes.forEach(([colorTheme, bgColor, brColor]) => {
```

If you go with the suggestion above this will work